src/ballet/sbpf/fd_sbpf_opcodes.h

```diff
@@ -199,8 +199,7 @@ static const uchar FD_SBPF_OP_JSLT32_REG =  FD_SBPF_DEFINE_NORM_INSTR(FD_SBPF_OP
 static const uchar FD_SBPF_OP_JSLE32_IMM =  FD_SBPF_DEFINE_NORM_INSTR(FD_SBPF_OPCODE_CLASS_JMP, FD_SBPF_OPCODE_JMP_OP_MODE_JSLE, FD_SBPF_OPCODE_SOURCE_MODE_IMM);
 static const uchar FD_SBPF_OP_JSLE32_REG =  FD_SBPF_DEFINE_NORM_INSTR(FD_SBPF_OPCODE_CLASS_JMP, FD_SBPF_OPCODE_JMP_OP_MODE_JSLE, FD_SBPF_OPCODE_SOURCE_MODE_REG);
 
-/* FIXME This should be FD_SBPF_OP_LDDW */
-static const uchar FD_SBPF_OP_LDQ =   FD_SBPF_DEFINE_MEM_INSTR(FD_SBPF_OPCODE_CLASS_LD,   FD_SBPF_OPCODE_ADDR_MODE_IMM,  FD_SBPF_OPCODE_SIZE_MODE_QUAD);
+static const uchar FD_SBPF_OP_LDDW =  FD_SBPF_DEFINE_MEM_INSTR(FD_SBPF_OPCODE_CLASS_LD,   FD_SBPF_OPCODE_ADDR_MODE_IMM,  FD_SBPF_OPCODE_SIZE_MODE_QUAD);
```

Comment:
You should rename FD_SBPF_OPCODE_SIZE_MODE_QUAD to FD_SBPF_OPCODE_SIZE_MODE_DOUB or something then